src/ceph-daemon

```diff
@@ -1051,32 +1141,35 @@ def command_rm_cluster():
 
     unit_name = 'ceph-%s.target' % args.fsid
     try:
-        subprocess.check_output(['systemctl', 'stop', unit_name])
-        subprocess.check_output(['systemctl', 'disable', unit_name])
-    except subprocess.CalledProcessError:
+        call_throws(['systemctl', 'stop', unit_name],
+                    verbose_on_failure=False)
+        call_throws(['systemctl', 'disable', unit_name],
+                    verbose_on_failure=False)
+    except RuntimeError:
         pass
 
     slice_name = 'system-%s.slice' % (
         ('ceph-%s' % args.fsid).replace('-', '\\x2d'))
     try:
-        subprocess.check_output(['systemctl', 'stop', slice_name])
-    except subprocess.CalledProcessError:
+        call_throws(['systemctl', 'stop', slice_name],
```

Comment:
since you're discarding the RuntimeError from this and the other stop commands, you could use call() instead of call_throws() and remove the exception handling here

Reply:
fixed!